js/app/components/login/login.tsx

```diff
@@ -86,51 +109,73 @@ export default function Login() {
 
   return (
     <KeyboardAvoidingView style={{ flex: 1 }} behavior="padding">
-      <View
-        f={1}
-        jc="center"
-        ai="center"
-        backgroundColor="$gray1"
-        padding="$4"
-        width="100%"
-        maxWidth={800}
-        marginHorizontal="auto"
+      <Form
+        flex={1}
+        onSubmit={async () => {
+          await dispatch(login(handle));
+        }}
       >
-        <Form
-          width="100%"
-          maxWidth={800}
+        <View
+          f={1}
           jc="center"
           ai="center"
-          onSubmit={async () => {
-            await dispatch(login(handle));
-          }}
+          padding="$4"
+          width="100%"
+          marginHorizontal="auto"
         >
-          <H3>Log in with ATProto | Bluesky</H3>
-          <H5 alignSelf="flex-start">Handle:</H5>
-          <Input
+          <YStack
+            px="$6"
+            py="$6"
+            br="$4"
+            backgroundColor="$color2"
             width="100%"
-            placeholder="example.bsky.social"
-            value={handle}
-            onChangeText={(text) => setHandle(text)}
-            keyboardType="url"
-            autoCapitalize="none"
-            autoComplete="off"
-            autoCorrect={false}
-          />
-          <Form.Trigger asChild>
-            <Button
-              width="100%"
-              margin="$4"
-              backgroundColor="$accentColor"
-              disabled={loginState.loading}
-            >
-              <Text>
-                {loginState.loading ? <Spinner /> : `Log in with ATProto`}
+            maxWidth={600}
+            gap="$2"
+          >
+            <Text fontSize="$9" fontWeight="200">
+              Log in
+            </Text>
+            <Text color="$color11">
+              Sign in using your handle on the Atmosphere (e.g. your Bluesky
```

Comment:
```suggestion
              Sign in using your handle on the AT Protocol (e.g. your Bluesky
```

If you can think of somewhere for that link to go we can do that too lol